Library/Formula/conoha-iso.rb

```diff
@@ -0,0 +1,34 @@
+require "language/go"
+
+class ConohaIso < Formula
+  desc "Command wrapper to use ISO of ConoHa."
+  homepage "https://github.com/hironobu-s/conoha-iso"
+  version "0.2.1"
+  url "https://github.com/hironobu-s/conoha-iso/archive/current.tar.gz"
+  sha256 "a922d949295ca05f8b3267df471e56bc36590aa9ceacb4ad0d972abdc8192b0a"
+
+  depends_on "go"  => :build
+
+  go_resource "github.com/Sirupsen/logrus" do
+    url "https://github.com/Sirupsen/logrus.git", :revision => "418b41d23a1bf978c06faea5313ba194650ac088"
+  end
+
+  go_resource "github.com/codegangsta/cli" do
+    url "https://github.com/codegangsta/cli.git", :revision => "565493f259bf868adb54d45d5f4c68d405117adf"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/hironobu-s/"
+    ln_s buildpath, buildpath/"src/github.com/hironobu-s/conoha-iso"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd buildpath do
+        system "go", "build", "-o", bin/"conoha-iso", "run.go"
+    end
+  end
+
+  test do
+    assert_match /conoha-iso version 0.1/, shell_output("#{bin}/conoha-iso --version")
```

Comment:
Could you apply a patch that fixes this? I suspect we'll get confused bug reports otherwise.